monitoring/remote_monitoring_demo/json_files/accelerator/exps/no_gpu_exp.json

```diff
@@ -0,0 +1,27 @@
+[{
```

Comment:
Do we need no_gpu_exp?  This is the container exp which is already covered in remote monitoring demo.